src/Cairo.jl

```diff
@@ -311,10 +323,10 @@ end
 
 function write_to_png(surface::CairoSurface, filename::AbstractString)
     ccall((:cairo_surface_write_to_png,_jl_libcairo), Void,
-          (Ptr{UInt8},Ptr{UInt8}), surface.ptr, bytestring(filename))
+          (Ptr{UInt8},Ptr{UInt8}), surface.ptr, @compat(String(filename)))
 end
 
-writemime(io::IO, ::MIME"image/png", surface::CairoSurface) =
+@compat Base.show(io::IO, ::MIME"image/png", surface::CairoSurface) =
```

Comment:
Looks like AppVeyor is now running on a JuliaGraphics account instead of my personal account, great. Could you (or someone else? @simondanisch maybe?) grant me collaborator access to that appveyor account? It would be good to enable a daily scheduled build here to catch potential WinRPM issues triggered by upstream changes. Also the readme appveyor badge needs updating.